style.css

```diff
@@ -0,0 +1,104 @@
+.ticket-container {
+    margin-top: 50px;
+    margin-bottom: 50px;
+    width: 782px;
+    height: 273px;
+    background: #EFC684;
+    border-radius: 10px;
+    display: flex;
+    font-family: Roboto;
+}
+
+.ticket-main {
+    width: 289px;
+}
+
+.ticket-text {
+    width: 206px;
+    height: 128px;
+    font-size: 55px;
+    margin-top: 26px;
+    margin-left: 20px;
+}
+
+.ticket-details {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 63px;
```

Comment:
When using margin or padding make sure that the values ar multiplication of the same number, usually it is a multiplication of 4px. It makes elements positioning and aligning a lot easier